src/executor/runContract.ts

```diff
@@ -96,7 +92,7 @@ export async function runContract(args: {
     return await executeContract({
         code: args.code,
         dataCell: args.data,
-        stack: [],
+        stack,
```

Comment:
I noticed that the stack is not passed to the executeContract method. I fixed this, clarify if it is needed.